lifecycle/cli.go

```diff
@@ -296,6 +309,22 @@ func export(c *cli.Context) error {
 	return commands.Exec(exportCmd)
 }
 
+func releaseBundleImport(c *cli.Context) error {
```

Comment:
Assert that exactly one argument was provided and check if the help is requested.
Missing on the export function too. Something like this:

```
	if show, err := cliutils.ShowCmdHelpIfNeeded(c, c.Args()); show || err != nil {
		return err
	}

	if c.NArg() != 1 {
		return cliutils.WrongNumberOfArgumentsHandler(c)
	}